deploy/packaging/petasos_spruce.yaml

```diff
@@ -294,3 +294,15 @@ service:
         # (Optional) defaults to empty struct
         # queryOptions:
         #   useCache: true
+
+# tracing provides configuration for OpenTelemetry
+tracing:
+  # provider is the provider name. Currently, stdout, jaegar and zipkin are supported.
+  provider: (( grab $TRACING_PROVIDER_NAME || "stdout" ))
+
+  # skipTraceExport only applies when provider is stdout. Set skipTraceExport to true
+  # so that trace information is not written to stdout.
+  skipTraceExport: true
+
+  # endpoint is where trace information should be routed. Applies to zipkin and jaegar.
+  endpoint: (( grab $TRACING_PROVIDER_ENDPOINT || "http://localhost:9411/api/v2/spans" ))
```

Comment:
this is good. however, since this is docker localhost is referring to the petasos container which I don't think we want. Maybe `zipkin`, `jaegar` or `tracehander`. It should be the container name that that will show up in the docker compose for example [caduceus](https://github.com/xmidt-org/xmidt/blob/e10ab4c61ab0cfb75b35be2486d9cfbddb4b1d09/deploy/docker-compose/docker-compose.yml#L3).

Reply:
Hi @kcajmagic  endpoint won't be used as we have given the provider as `stdout`.

Reply:
Hey Sachin, Jack is right. I think we should change this to `zipkin` especially since I'm adding zipkin to visualize the traces in our docker compose cluster https://github.com/xmidt-org/xmidt/pull/47/files